packages/android_intent/test/android_intent_test.dart

```diff
@@ -43,6 +43,14 @@ void main() {
       }));
     });
 
+    test('asserts on missing action & component', () {
```

Comment:
Can you name this test based on the expectation that is violated? For example: Raises an error if neither an action nor a component is provided.

Also, can you add 2 tests that demonstrate the usage of just a single property? For example:

`test('Can send Intent with an action and no component', () {..});`

and

`test('Can send Intent with a component and no action', () {...});`

Those 2 new tests will leave breadcrumbs for other developers to understand the intended behavior.

Reply:
Followed your suggestion and indeed it's a lot clearer now.